pages/product-page.html

```diff
@@ -0,0 +1,349 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Shoe Shoppers| Products</title>
+    <link rel="stylesheet" href="https://neos-ui.netlify.app/components/neos-ui.css" />
+    <link rel="stylesheet" href="/styles/product-page.css">
+    <script src="https://kit.fontawesome.com/0b2d218a52.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+    <!----------------------------navbar starts--------------------------->
+    <div class="nav-container">
+        <nav class="nav-heading">
+            <a href="/" class="brand">
+                <h1 class="brand-name">Shoe-Shoppers</h1>
+            </a>
+            <div class="nav-search">
+                <input type="text" class="search-input-box" placeholder=" Type to search">
+                <div class="search-button">
+                    <a href="#"><i class="fa fa-search"></i></a>
+                </div>
+            </div>
+            <div class="link-container">
+
```

Comment:
removing the white Spaces! Else Everything looks nice to me, good to merger after changes